pyscript.core/tests/integration/test_01_basic.py

```diff
@@ -159,7 +158,6 @@ def test_execution_in_order(self):
             "four",
         ]
 
-    @skip_worker("NEXT: something very weird happens here")
```

Comment:
nice!
Maybe we can rewrite the test to use only two tags instead of 4?
```html
<script type="py">
    import js
    js.console.log("A", 1<2, 1>2)
    js.console.log("B <div></div>")
</script>
<py-script>
    import js
    js.console.log("C", 1<2, 1>2)
    js.console.log("D <div></div>")
</py-script>
```

This should be a bit faster in the `worker` base because it spawns only two pyodides instead of 4.
It was my original plan but I had to revert because of this bug.

Reply:
done ... let's wait for CI to be green :crossed_fingers: then I'll merge with this change in: 2+ secs saved, I am all in for it!

Reply:
this side-effected on *polyscript* ... fixed and reused same logic in here ... let's wait again for CI